02_configure_host.sh

```diff
@@ -169,3 +169,20 @@ if [ "$MANAGE_BR_BRIDGE" == "y" ] ; then
     sudo systemctl restart NetworkManager
   fi
 fi
+
+# Create openstack clouds.yaml
+if [[ ! -f "$OPENSTACK_CONFIG" ]]
+then
+  mkdir -p "$(dirname "$OPENSTACK_CONFIG")"
+  echo "clouds:" > "$OPENSTACK_CONFIG"
+fi
+
+if ! grep -qi metal3 "$OPENSTACK_CONFIG"
+then
+  cat <<EOF >>"$OPENSTACK_CONFIG"
+  metal3:
+    auth_type: none
+    baremetal_endpoint_override: http://172.22.0.2:6385
+    baremetal_introspection_endpoint_override: http://172.22.0.2:5050
```

Comment:
This will only work with the bootstrap VM instance of ironic, in the existing clouds.yaml we have to "clouds" metal3-bootstrap and metal3, so we probably should reuse that I think?

It's not currently installed automatically, but we could do that and update the README?

Reply:
Oh I see, I didn’t realize we had one already  :man_facepalming: I only remembered seeing a an abandoned PR. 

This is problematic, I called the metal3-dev-env one metal3, but it uses the same IP as OpenShift’s bootstrap.

Reply:
Why is is problematic if both the name and the endpoints are the same?

Although if we want to be more explicit personally I'd be fine with renaming the dev-scripts ones, maybe `bootstrap` and `cluster` or something?

Reply:
Oh I see, the cluster is the metal3 one...

Reply:
Yea, exactly. I could make minikube take the .3 address so we could re-use the `metal3` name. But, eventually metal3 could deploy a real k8s cluster using minikube like a bootstrap, which means the .3 would be that cluster I guess. Then we're up to potentially 4 Ironics to talk to.

Would it be possible they could all end up being configured differently as well? Especially if we can customize IP's at some point. Maybe I'm overthinking things, but that's why I made them all separate